extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosInterface.java

```diff
@@ -178,7 +194,7 @@ BatchMediaItemResponse createPhotos(NewMediaItemUpload newMediaItemUpload)
   }
 
   private <T> T makeGetRequest(String url, Optional<Map<String, String>> parameters, Class<T> clazz)
-          throws IOException, InvalidTokenException, PermissionDeniedException {
+      throws IOException, InvalidTokenException, PermissionDeniedException, UploadErrorException {
```

Comment:
it seems wrong that all of these methods now have to throw UploadErrorException and handle them in the exporter when there is no upload happening here. Why does the makeGetRequest function have to throw that specific error?